### ISSUE TITLE
Should we remove kinto.core features unused in Kinto ?

### ISSUE THREAD

**Issue by leplatrem** [Kinto/kinto#765]
`kinto.core` is the toolkit leveraged by Kinto but that can be used to implement domain-specific APIs. It ships a lot of good practices by default.

Historically, `kinto.core` was _Cliquet_ a toolkit in a separate repo.
It has features that were implemented for the _readinglist_, _Syncto_ and _kinto_ projects.

But now it is the core of Kinto, should we keep core features that are unused ?

For example:
- [readonly fields in resources](https://github.com/Kinto/kinto/blob/master/kinto/core/resource/schema.py#L24-L29)
- [resources without permissions (records per user-id)](https://github.com/Kinto/kinto/blob/master/kinto/core/resource/__init__.py#L126)
- [private permission in authz policy](https://github.com/Kinto/kinto/blob/master/kinto/core/authorization.py#L14-L15)
- [newrelic monitoring agent](https://github.com/Kinto/kinto/blob/master/kinto/core/initialization.py#L289-L294)

In other words, should we get rid of these now, and re-introduce them the day we need them, or should we keep them ­— just in case ?

---

Quick poll: https://twitter.com/Natim/status/765884024934002690

**Comment by Natim** [comment]
> resources without permissions (records per user-id)

For this one in particular, we are using to store plugin settings ([kinto-webpush](https://github.com/Kinto/kinto-webpush/))

For most of these features I don't see the benefit of getting rid of them being really likely to need the feature back again in a near future.

**Comment by leplatrem** [comment]
Related https://github.com/Kinto/kinto/pull/763

**Comment by leplatrem** [comment]
BTW I'm +1 on removing readonly fields. /cc @gabisurita
